doc/Extensions/Applications.md

```diff
@@ -229,6 +230,28 @@ The application should be auto-discovered as described at the top of
 the page. If it is not, please follow the steps set out under `SNMP
 Extend` heading top of page.
 
+# backupninja
+
+A small shell script that reports status of last backupninja backup.
+
+## SNMP Extend
+
+1: Download the [backupninja
+script](https://github.com/librenms/librenms-agent/blob/master/snmp/backupninja.py)
+to `/etc/snmp/backupninja.py` on your backuped server.
+```
+wget https://raw.githubusercontent.com/librenms/librenms-agent/master/snmp/backupninja.py -O /etc/snmp/backupninja.py`
+```
+
+2: Edit your snmpd.conf file (usually `/etc/snmp/snmpd.conf`) and add:
+
+```
+extend .1.3.6.1.4.1.2021.220 backupninja /usr/bin/python /etc/snmp/backupninja.py
```

Comment:
don't use hardcoded oid numbers!
take a look on other Applications how it's made

used interpreter shouldn't be declarated